packages/components/src/internal/components/base/DragDropHandle.tsx

```diff
@@ -10,14 +8,15 @@ interface Props {
 export class DragDropHandle extends React.Component<Props, any> {
     render() {
         const { highlighted } = this.props;
-        const className =
-            highlighted !== undefined
-                ? classNames({
-                      'field-highlighted': highlighted,
-                      'field-not-highlighted': !highlighted,
```

Comment:
was `field-not-highlighted` just used to apply the gray color? I assume that isn't needed anymore

Reply:
Yeah, it just enforces the default look, it's not needed